test/extended/router/gatewayapicontroller.go

```diff
@@ -160,17 +163,32 @@ var _ = g.Describe("[sig-network-edge][OCPFeatureGate:GatewayAPIController][Feat
 
 			g.By("Deleting the OSSM Operator resources")
 
-			operator, err := operatorsv1.NewForConfigOrDie(oc.AsAdmin().UserConfig()).Operators().Get(context.Background(), serviceMeshOperatorName, metav1.GetOptions{})
+			client, err := dynamic.NewForConfig(oc.AdminConfig())
```

Comment:
nit: the client already exists in `oc.KubeFramework().DynamicClient` (see line 186 below)